forwards.h doesn't declare most useful classes

[`json/forwards.h`](https://github.com/open-source-parsers/jsoncpp/blob/master/include/json/forwards.h) forward declares the deprecated FastWriter and Reader, but not the non-deprecated replacements StreamWriter and CharReader.

I've tried to add declaration `class StreamWriter` and `class CharReader` in the `forwards.h` then  execute `make` command the result is：
`[ 11%] Building CXX object src/lib_json/CMakeFiles/jsoncpp_lib.dir/json_reader.cpp.o
In file included from /software/jsoncpp-1.8.4/include/json/features.h:10:0,
                        from /software/jsoncpp-1.8.4/include/json/reader.h:10,
                        from /software/jsoncpp-1.8.4/src/lib_json/json_reader.cpp:9:

/software/jsoncpp-1.8.4/include/json/forwards.h:21:7: error: multiple types in one declaration
 class Reader;
       ^
/software/jsoncpp-1.8.4/include/json/forwards.h:25:7: error: multiple types in one declaration
 class Features;
       ^
src/lib_json/CMakeFiles/jsoncpp_lib.dir/build.make:62: recipe for target 'src/lib_json/CMakeFiles/jsoncpp_lib.dir/json_reader.cpp.o' failed
make[2]: *** [src/lib_json/CMakeFiles/jsoncpp_lib.dir/json_reader.cpp.o] Error 1
CMakeFiles/Makefile2:114: recipe for target 'src/lib_json/CMakeFiles/jsoncpp_lib.dir/all' failed
make[1]: *** [src/lib_json/CMakeFiles/jsoncpp_lib.dir/all] Error 2
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2
`  But I don't know why the ***class Features***   multipled declared here. Is there anyone  could explain it?

--- a/include/json/forwards.h
+++ b/include/json/forwards.h
@@ -13,11 +13,17 @@
 namespace Json {
 
 // writer.h
+class StreamWriter;
+class StreamWriterBuilder;
+class Writer;
 class FastWriter;
 class StyledWriter;
+class StyledStreamWriter;
 
 // reader.h
 class Reader;
+class CharReader;
+class CharReaderBuilder;
 
 // features.h
 class Features;

